Gaijinizer patch/Map247.txt

```diff
@@ -113,9 +129,10 @@ are shared. Very confusing ...#MTLed
  
 区別ができるのは、目の色が変わった時のみ……
 > CONTEXT: Map247/events/5/pages/0/95/Dialogue/yuuma
-...... Well, tone and voice. Because it is \\N[1]
-itself...The distinction can be made only when
-the eye color changes ...#MTLed
+......Indeed, since the manner of speaking and the
+voice are also \\N[1]'s own...
+You can only tell them apart when the color of
+their eyes change......#Strate
 > END STRING
```

Comment:
Minor formatting:
```
...Indeed, since the manner of speaking and the
voice are also \\N[1]'s own, you can only tell
them apart when the color of their eyes change...#Strate
```